test/libsolidity/MemoryGuardTest.cpp

```diff
@@ -76,11 +76,18 @@ TestCase::TestResult MemoryGuardTest::run(std::ostream& _stream, std::string con
 		}
 
 		auto handleObject = [&](std::string const& _kind, Object const& _object) {
-			m_obtainedResult += contractName + "(" + _kind + ") " + (findFunctionCalls(
-				_object.code()->root(),
-				"memoryguard",
-				yulStack.dialect()
-			).empty() ? "false" : "true") + "\n";
+			auto const memoryGuardHandle = yulStack.dialect().findBuiltin("memoryguard");
+			m_obtainedResult += contractName + "(" + _kind + ") ";
+			if (memoryGuardHandle &&
+				!findFunctionCalls(
+					_object.code()->root(),
+					BuiltinName{nullptr, *memoryGuardHandle},
+					yulStack.dialect()
+				).empty()
+			)
```

Comment:
```suggestion
			if (
				memoryGuardHandle &&
				!findFunctionCalls(
					_object.code()->root(),
					BuiltinName{nullptr, *memoryGuardHandle},
					yulStack.dialect()
				).empty()
			)
```
But it you can make it shorter if you split the parameter list rather than the condition:
```suggestion
			if (memoryGuardHandle && !findFunctionCalls(
				_object.code()->root(),
				BuiltinName{nullptr, *memoryGuardHandle},
				yulStack.dialect()
			).empty())
```